packages/@orbit/record-cache/src/live-query/utils.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  Record,
+  cloneRecordIdentity,
+  RecordIdentity,
+  RecordOperation
+} from '@orbit/data';
+
+export interface RecordChange extends RecordIdentity {
+  keys: string[];
+  attributes: string[];
+  relationships: string[];
+  remove: boolean;
+}
+
+export function recordOperationChange(
+  operation: RecordOperation
+): RecordChange {
+  const record = operation.record as Record;
+  const change: RecordChange = {
+    ...cloneRecordIdentity(record),
+    remove: false,
+    keys: [],
+    attributes: [],
+    relationships: []
+  };
+
+  switch (operation.op) {
+    case 'addRecord':
+    case 'updateRecord':
```

Comment:
Check for `record.meta` and `record.links`.

Reply:
I am realizing that current implementation of change matching is not using `attributes` or `keys` (nor `meta` or `links`). Right now the algorithm is just to check if the `identity` or relationship `identity` is the same, we consider change relevant. So in practice we could only expose `identity`, `relationships`, and `removed` in the `RecordChange` interface. But we also may improuve the algorith in the future and use extra information.